frontend/src/components/HamburgerMenu.tsx

```diff
@@ -302,15 +307,25 @@ export default function HamburgerMenu({
           <RouterLink to="../live/">
             <MenuItem icon={<Icon as={FaHome} />}>Home</MenuItem>
           </RouterLink>
-          <MenuItem
-            onClick={() => {
-              onProfileOpen();
-            }}
-            cursor="pointer"
-            icon={<Icon as={SettingsIcon} />}
-          >
-            Settings
-          </MenuItem>
+          {isSongbookOwner && (
+            <>
+              <MenuItem
+                onClick={() => {
+                  onSettingsOpen();
+                }}
+                cursor="pointer"
+                icon={<Icon as={SettingsIcon} />}
+              >
+                Settings
```

Comment:
This is a product topic - I'm not sure about replacing the existing settings button with this. Two thoughts:
1) I think overloading the term "settings" is a little confusing (i.e. settings meaning something different depending on what page you're on
2) You can't access the user settings from the current song view now

Thoughts?

Reply:
Hamburgers can never have enough options.